changelogs/fragments/vmware_host_firewall_manager-prepare-all_ip-transition.yaml

```
@@ -0,0 +1,5 @@
+---
+minor_changes:
```

```suggestion
deprecated_features:
```
Then this will get included in the porting guide.